source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m

```diff
@@ -315,7 +332,19 @@ - (void)didFetchUserResponses:(ACOAdaptiveCard *)card action:(ACOBaseActionEleme
             [self reloadRowsAtChatWindowsWithIndexPaths:self.chatWindow.indexPathsForSelectedRows];
         }
         NSString *str = [NSString stringWithFormat:@"{\n%@\n}", [fetchedInputList componentsJoinedByString:@",\n"]];
-        [self presentViewController:[self createAlertController:@"user response fetched" message:str] animated:YES completion:nil];
+        
+        // if the app is being tested we set the result in the uilabel, otherwise
+        // we show the label in a popup
```

Comment:
Is it not possible to access text from the alert during UI tests?

Reply:
according to sack overflow, it's not possible
https://stackoverflow.com/questions/33667861/ios-ui-test-how-to-get-message-of-uialertcontroller

Reply:
@almedina-ms 
i like your new tests.
i was wondering if you could implement input delegate directly at the ui unit test. currently the delegate is in ViewController.
so for example, submit is pressed, you get the chance to intercept the input there in the ui test, so you don' t have to create another input element.s